infos/src/main.rs

```diff
@@ -425,11 +423,10 @@ fn biggest_value_sizes(index: &Index, rtxn: &heed::RoTxn, limit: usize) -> anyho
             heap.pop();
         }
 
-        if let Some(documents_ids) = main.get::<_, Str, ByteSlice>(rtxn, "documents-ids")? {
-            heap.push(Reverse((documents_ids.len(), format!("documents-ids"), main_name)));
-            if heap.len() > limit {
-                heap.pop();
-            }
+        let documents_ids = index.documents_ids(rtxn)?;
+        heap.push(Reverse((documents_ids.len() as usize, format!("documents-ids"), main_name)));
```

Comment:
```suggestion
        heap.push(Reverse((documents_ids.len() as usize, "documents-ids".to_string(), main_name)));
```

Since we don't format anything it's better to do a `to_string` I guess?